templates/telemetry/crash/crash.4.schema.json

```diff
@@ -15,6 +15,9 @@
           "type": "string",
           "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
         },
+        "crashId": {
+          "type": "string"
```

Comment:
Is crashId expected to be a UUID? Should we enforce that with a pattern?

Reply:
Is crashId always present, or is it potentially null?

Reply:
Tests for these cases would be good.

Reply:
It does appear to be a UUID, but I'm not sure if we would start to reject any pings if we enforced the validation. I'll add this and check the integration report. 

The documentation for thing ping marks the field as optional: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/data/crash-ping.html

The field is already included as part of the passing document here: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/e8226ee26926df54fb641cee02c4e91b0d346a5b/validation/telemetry/crash.4.sample.pass.json#L188

Reply:
It looks like there's a small increase in the number of rejected pings, based on the document sample:

```diff
--- 
+++ 
@@ -168,7 +168,7 @@
         "error_rate": 0.0
     },
     "telemetry.crash.4": {
-        "error_rate": 1.9
+        "error_rate": 2.0
     },
     "telemetry.deletion.4": {
         "error_rate": 0.0
```